app/views/layouts/minimal.erb

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
@emcelroy -- This can probably be removed it was from the previous approach where the plugin was rendered through the ERB templates.  I will see if I can back out these chaangse.